docs/minio-admin-complete-guide.md

```diff
@@ -10,6 +10,7 @@ policy       manage canned policies
 credential   change admin server access and secret keys
 config       manage configuration file
 heal         heal disks, buckets and objects on minio server
+top          command similar to top in Linux
```

Comment:
`sh` in comment block is causing color coding of the words `command` and `in`

Reply:
wordings have changed